app/assets/stylesheets/components/_option-select.scss

```diff
@@ -176,16 +97,91 @@
   fill: currentColor;
 }
 
-.js-enabled .app-c-option-select__icon--up {
-  display: block;
+.app-c-option-select__container {
+  position: relative;
+  height: 200px;
+  background-color: $page-colour;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.app-c-option-select__container-inner {
+  padding: $gutter-one-quarter;
 }
 
-.js-closed {
+.app-c-option-select__filter {
+  display: none;
+
+  .govuk-form-group {
+    margin-bottom: $gutter-one-third;
+  }
+}
+
+.js-enabled {
+  .app-c-option-select {
+    padding: 0 0 1px;
```

Comment:
why is 1px needed?

Reply:
I think it overrides the non-JS behaviour, which is to have much more padding for some reason, try toggling JS in your browser to see the difference (this looks like my change but I was just moving it around).